client-app/shared/account/composables/useUser.ts

```diff
@@ -47,39 +48,77 @@ const isCorporateMember = computed<boolean>(() => !!user.value?.contact?.organiz
 const organization = eagerComputed<Organization | null>(() => user.value?.contact?.organizations?.items?.[0] ?? null);
 const operator = computed<UserType | null>(() => user.value?.operator ?? null);
 
+interface IPasswordExpirationEntry {
+  userId: string;
+  date: Date;
+}
+
 export function useUser() {
   const broadcast = useBroadcast();
   const { innerFetch } = useFetch();
   const { openPopup, closePopup } = usePopup();
 
-  const changePasswordReminderDate: RemovableRef<Date | null> = useStorage("vcst-password-expire-reminder-date", null);
+  const changePasswordReminderDates: RemovableRef<string | null> = useLocalStorage(
```

Comment:
Please replace with with `useLocalStorage<IPasswordExpirationEntry[]>` so you will not need to manually parse it and cast types